Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["field", "insertion", "plonky2", "starky", "system_zero", "util", "waksman", "ecdsa"]
+members = ["field", "insertion", "plonky2", "starky", "system_zero", "util", "waksman", "ecdsa", "plonky2_u32"]
```

Comment:
nit: existing convention is to include `plonky2_` in the crate name but not the directory

Reply:
For some reason I felt that having a directory called `u32` would be a bit weird, but I agree we should stick to the convention 👍